db/sql/00_msar.sql

```diff
@@ -1037,6 +1037,25 @@ WHERE role_data.name = rolename;
 $$ LANGUAGE SQL STABLE;
 
 
+CREATE OR REPLACE FUNCTION
+msar.get_current_role() RETURNS TEXT AS $$/*
+Returns a JSON object describing the current_role and the parent role(s) it inherits from.
+*/
+SELECT jsonb_build_object(
+  'current_role', msar.get_role(current_role),
+  'parent_roles', array_remove(
+    array_agg(
+      CASE WHEN pg_has_role(role_data.name, current_role, 'MEMBER')
```

Comment:
I think we should make this check on `'USAGE'`, since we currently have no way to `SET ROLE`, and showing roles without `'USAGE'` would therefore confuse users. @pavish What do you think?